src/solvers/petsc_linear_solver.C

```diff
@@ -140,18 +148,17 @@ void PetscLinearSolver<T>::init (const char * name)
       PetscErrorCode ierr=0;
 
       // Can't use libmesh_make_unique with custom deleter
```

Comment:
Does this comment still make any sense here?

Reply:
No, that should have been removed when I dropped `petsc_unique_ptr`.